ecosystem/python/sdk/aptos_sdk/account.py

```diff
@@ -79,6 +80,37 @@ def public_key(self) -> ed25519.PublicKey:
         return self.private_key.public_key()
 
 
+class RotationProofChallenge:
```

Comment:
so I was suggesting that we emit out a chunk of bytes from say https://github.com/aptos-labs/aptos-core/blob/58214fc22ec00ebaeda467416e27574cdc6eb01d/aptos-move/e2e-move-tests/src/tests/rotate_auth_key.rs#L162

and dumping this to hex and importing that hex string here... that way we can be confident that it works as expected.

Reply:
@davidiw how does this look?

https://github.com/alnoki/aptos-core/blob/dc8ac59fe89ecf04f2b6187a7e2ae83179cde23a/ecosystem/python/sdk/aptos_sdk/account.py#L131-L160

The classes have been verified e2e on devnet using the example script broken down in the tutorial, so I just emitted some bytes and made a test for it.

Reply:
great